movies/src/resources/doc.jsonld

```diff
@@ -6,10 +6,52 @@
   "hydra:description": "This API shows how a basic CRUD (Create, Read, Update, Delete) API can be designed using Hydra Core Vocabulary.",
   "hydra:entrypoint": "/",
   "hydra:supportedClass": [
+    {
+      "@id": "schema:EntryPoint",
+      "@type": "hydra:Class",
+      "hydra:title": "API entrypoint",
+      "hydra:description": "Every capability of the API can be reached from here by following links.",
+      "hydra:supportedProperty": [
+        {
+          "hydra:property": "hydra:collection",
```

Comment:
This is one of my problems with `hydra:collection`. It's an odd one, which can implicitly appear on any resource but the API author should not be forced to include it in all classes. Doesn't make sense IMO.

Nor is any kind of subclassing/inheritance an option here.

If `hydra:collection` is here to stay I think it should remain hidden and implicit. The [entrypoint recipe](https://hydra-cookbook.netlify.com/api-documentation/1.entry-point.html) indeed shows how the collections are part of the client's API.

Thus, I propose that we remove it from the api doc and make clear in the spec that clients should be responsible for handling them on any resource as they can appear anywhere.

Reply:
@tpluscode I added it to the API doc because your generic app did not show it, if it is missing ;) I removed it from the docs, but I guess your client needs updating then

Reply:
Haha, indeed! Alcaeus does not understand `hydra:collection` currently but I will add it